airflow-core/src/airflow/api_fastapi/auth/managers/base_auth_manager.py

```diff
@@ -73,6 +73,7 @@
 
 
 COOKIE_NAME_JWT_TOKEN = "_token"
+COOKIE_NAME_REFRESH_JWT_TOKEN = "_refresh_token"
```

Comment:
No need to use a different cookie name, we should use the existing one `COOKIE_NAME_JWT_TOKEN`. Then we need to modify the UI to override the token when provided

Reply:
Reverted these pieces and used the generic, current key